dsa/validate.go

```diff
@@ -0,0 +1,51 @@
+package dsa
+
+import (
+	"github.com/prebid/prebid-server/v2/exchange/entities"
+	"github.com/prebid/prebid-server/v2/openrtb_ext"
+
+	"github.com/buger/jsonparser"
+)
+
+const (
+	// Required - bid responses without DSA object will not be accepted
+	Required = 2
+	// RequiredOnlinePlatform - bid responses without DSA object will not be accepted, Publisher is an Online Platform
+	RequiredOnlinePlatform = 3
+)
+
+// Validate determines whether a given bid is valid from a DSA perspective.
+// A bid is considered valid unless the bid request indicates that a DSA object is required
+// in bid responses and it the object happens to be missing from the specified bid.
```

Comment:
Nitpick: I think the word it can be removed from this comment so it'll look like:

`// in bid responses and the object happens to be missing from the specified bid.`